jenkins-pipelines/perf-regression-cdc-write-60min.jenkinsfile

```diff
@@ -0,0 +1,16 @@
+#!groovy
```

Comment:
the other file had latency in it's name, what's this one ? throughput ?

Reply:
yes, this is throughput. Change name